build.py

```diff
@@ -659,6 +659,8 @@ def run_ci_historic_benchmark():
         for dp, _, filenames in os.walk(samples_src)
         for f in filenames
         if f.endswith(".ipynb")
+        # Skip all Cirq sample notebooks under the cirq_samples folder
```

Comment:
Can you provide a reason why we skip them.

Reply:
cirq is not installed in the build machines. Missing dependencies seems to be the reason for why the other notebooks were skipped.

Reply:
done